src/main/scala/is/hail/methods/LinearRegression.scala

```diff
@@ -15,7 +15,7 @@ object LinearRegression {
     ("tstat", TDouble),
     ("pval", TDouble))
 
-  def apply(vds: VariantDataset, ySA: String, covSA: Array[String], root: String, useDosages: Boolean, minAC: Int, minAF: Double): VariantDataset = {
+  def apply(vds: VariantDataset, ySA: String, covSA: Array[String], root: String, useDosages: Boolean, useDominance: Boolean, useNormalise: Boolean, minAC: Int, minAF: Double): VariantDataset = {
```

Comment:
We should probably spell this "normalize" since we use American spellings everywhere else.